pkg/asset/manifests/tectonic.go

```diff
@@ -61,29 +56,23 @@ func (t *tectonic) Generate(dependencies map[asset.Asset]*asset.State) (*asset.S
-		"ingress/pull.json":                                           applyTemplateData(ingress.Pull, templateData),
-		"ingress/cluster-config.yaml":                                 applyTemplateData(ingress.ClusterConfig, templateData),
-
-		// constant files
-		"security/priviledged-scc-tectonic.yaml":                 []byte(security.PriviledgedSccTectonic),
-		"rbac/role-admin.yaml":                                   []byte(rbac.RoleAdmin),
-		"rbac/binding-admin.yaml":                                []byte(rbac.BindingAdmin),
-		"rbac/binding-discovery.yaml":                            []byte(rbac.BindingDiscovery),
-		"rbac/role-user.yaml":                                    []byte(rbac.RoleUser),
-		"updater/migration-status-kind.yaml":                     []byte(updater.MigrationStatusKind),
-		"updater/app_versions/app-version-kube-addon.yaml":       []byte(appversions.AppVersionKubeAddon),
-		"updater/app_versions/app-version-tectonic-ingress.yaml": []byte(appversions.AppVersionTectonicIngress),
-		"updater/app_versions/app-version-kube-core.yaml":        []byte(appversions.AppVersionKubeCore),
-		"updater/app-version-kind.yaml":                          []byte(updater.AppVersionKind),
-		"ingress/svc-account.yaml":                               []byte(ingress.SvcAccount),
+		"99_binding-discovery.yaml":                  []byte(content.BindingDiscovery),
+		"99_kube-addon-00-appversion.yaml":           []byte(content.AppVersionKubeAddon),
+		"99_kube-addon-01-operator.yaml":             applyTemplateData(content.KubeAddonOperator, templateData),
+		"99_kube-core-00-appversion.yaml":            []byte(content.AppVersionKubeCore),
+		"99_kube-core-00-operator.yaml":              applyTemplateData(content.KubeCoreOperator, templateData),
+		"99_role-admin.yaml":                         []byte(content.RoleAdmin),
+		"99_role-user.yaml":                          []byte(content.RoleUser),
+		"99_tectonic-ingress-00-appversion.yaml":     []byte(content.AppVersionTectonicIngress),
+		"99_tectonic-ingress-01-cluster-config.yaml": applyTemplateData(content.ClusterConfigTectonicIngress, templateData),
+		"99_tectonic-ingress-02-tls.yaml":            applyTemplateData(content.TLSTectonicIngress, templateData),
+		"99_tectonic-ingress-03-pull.yaml":           applyTemplateData(content.PullTectonicIngress, templateData),
+		"99_tectonic-ingress-04-svc-account.yaml":    []byte(content.SvcAccountTectonicIngress),
+		"99_tectonic-ingress-05-operator.yaml":       applyTemplateData(content.TectonicIngressControllerOperator, templateData),
+		"99_tectonic-system-00-binding-admin.yaml":   []byte(content.BindingAdmin),
+		"99_tectonic-system-01-ca-cert.yaml":         applyTemplateData(content.CaCertTectonicSystem, templateData),
+		"99_tectonic-system-02-privileged-scc.yaml":  []byte(content.PriviledgedSccTectonicSystem),
+		"99_tectonic-system-03-pull.yaml":            applyTemplateData(content.PullTectonicSystem, templateData),
```

Comment:
json, not yaml

Reply:
> json, not yaml

JSON is a subset of YAML, so a YAML parser (triggered by the `.yaml` suffix?) should have no trouble with JSON content read from that file.  Do we have JSON-only tooling that would care about distinguishing JSON files from YAML-with-features-outside-of-JSON files?